x/data/server/features/register_resolver.feature

```diff
@@ -0,0 +1,30 @@
+Feature: Register Resolver
+
+  Scenario: data is registered to the resolver when the data has been anchored
+    Given a valid content hash
+    And alice has anchored the data
+    And alice has defined a resolver
+    When alice attempts to register the data to the resolver
+    Then an error of ""
+    And the data resolver entry exists
+
+  Scenario: data is anchored and registered to the resolver when the data has not been anchored
+    Given a valid content hash
+    And alice has defined a resolver
+    When alice attempts to register the data to the resolver
+    Then an error of ""
+    And the data anchor entry exists
+    And the data resolver entry exists
```

Comment:
any way we can test one thing per scenario and have fewer ands?

Reply:
duplicate